analyse_performance.py

```
@@ -0,0 +1,490 @@
+
+    # Define the relevant file names which must be deleted
+    FILE_TEMPLATES = {"walls": "walls.{}",
+                      "tflow": "tflow.{}",
+                      "tdat": "tdat.{}"}
+
+
+    def __init__(self,
+                 beta_tip: list[float] | np.typing.NDArray[np.floating],
+                 rps_range: list[float] | np.typing.NDArray[np.floating],
+                 verbose: bool = True,
+                 **kwargs) -> None:
```

Ruff (0.14.7)

Unused method argument: `kwargs`

(ARG002)